src/pf-accordion/pf-accordion.component.js

```diff
@@ -0,0 +1,335 @@
+          for (let i = 0; i < mutation.addedNodes.length; i++) {
+            let node = mutation.addedNodes[i];
+            // only check for element nodes
+            if (1 === node.nodeType && 'pf-accordion-panel' === node.tagName.toLowerCase()) {
+              let panel = node.querySelector('pf-accordion-template');
+              if (null !== panel) {
+                this._checkAndAddPanel(panel);
+              }
+            }
+          }
+
+          // handle removal of panels
+          for (let i = 0; i < mutation.removedNodes.length; i++) {
+            let node = mutation.removedNodes[i];
+            // only check for element nodes
+            if (1 === node.nodeType && 'pf-accordion-panel' === node.tagName.toLowerCase()) {
+              let panel = node.querySelector('pf-accordion-template');
+              if (null !== panel) {
+                this._checkAndRemovePanel(panel);
+              }
+            }
+          }
+
+          // fixed height needs to be recalculated on DOM initialization
+          if (this.hasAttribute('fixedheight')) {
+            this._setFixedHeight();
+          }
+        }
+      }
+    });
+
+    this._observer.observe(this, {
+      childList: true
+    });
+
+    if (this.hasAttribute('fixedheight')) {
+      // _initialized is raised after _initFixedHeight
+      this._initFixedHeight();
+    } else {
+      this._initialized = true;
+      this.dispatchEvent(new Event('pf-accordion.initialized'));
+    }
+  }
+
+  /**
+   * Check and add panel if it is initialized
+   * @private
+   * @param {PfAccordionTemplate} panel the toggle pane;
+   */
+  _checkAndAddPanel(panel) {
+    if (panel._initialized) {
+      if (panel.open) {
+        this._openPanels.push(panel);
+      }
+    } else {
+      pfUtil.once(panel, 'pf-accordion.initialized', () => {
+        if (panel.open) {
+          this._openPanels.push(panel);
+        }
+      });
+    }
+  }
+
+  /**
+   * Check and remove a panel if it is in the array of open panels
+   * @private
+   * @param {PfAccordionTemplate} panel the toggle pane;
+   */
+  _checkAndRemovePanel(panel) {
+    let index = this._openPanels.indexOf(panel);
+    if (index > -1) {
+      this._openPanels.splice(index, 1);
+    }
+  }
+
+  /**
+   * Handle bubbled pf-accordion-collapsing on accordion
+   * @param {Event} e event
+   * @private
+   */
+  _handlePanelHidden(e) {
+    let index = this._openPanels.indexOf(e.target);
+    if (index > -1) {
+      this._openPanels.splice(index, 1);
+    }
+  }
+
+  /**
+   * Handle bubbled pf-accordion-expanding on accordion
+   * @param {Event} e event
+   * @private
+   */
+  _handlePanelShown(e) {
+    let panel;
+    while ((panel = this._openPanels.shift())) {
+      panel.open = false;
+    }
+    this._openPanels.push(e.target);
+  }
+
+  /**
+   * Called when the element is removed from the DOM
+   */
+  disconnectedCallback() {
+    this._observer.disconnect();
```

Comment:
this._observer will be undefined on pf-accordion being removed. So just remove this sentence.